recipes/renode-cli/helpers/renode_build_with_dotnet.ps1

```diff
@@ -0,0 +1,71 @@
+param (
+    [string]$framework_version
+)
+
+$SRC_DIR = Resolve-Path "$Env:SRC_DIR"
+
+$OUTPUT_DIRECTORY = "$SRC_DIR\output"
+$CONFIGURATION = "Release"
+$BUILD_PLATFORM = "Any CPU"
+$HEADLESS = $true
+$NET = $true
+$TFM = "net$framework_version-windows"
+$HOST_ARCH = "i386"
+$CMAKE_COMMON = ""
```

Comment:
See my previous comments about reducing usage of variables - if we simplify this logic enough, we won't even need a wrapper anymore.